server/services/prisonerProfileService.test.ts

```diff
@@ -25,6 +26,23 @@ describe('Prisoner profile service', () => {
     })
 
     it('Retieves and processes data for prisoner profile with visit balances', async () => {
```

Comment:
I know it's not in the change but spelling 😁

Reply:
well spotted 🤦‍♂️!